pysatNASA/constellations/icon.py

```diff
@@ -1,19 +1,15 @@
-import pysat
 """
-Creates a constellation from NASA ICON instrumentation
+Creates a constellation from NASA the ICON satellite platform
 """
 
-# FIXME: syntax has changed since this was written.  Not imported
-ivm = pysat.Instrument('icon', 'ivm', sat_id='a', tag='level_2',
-                       clean_level='clean', update_files=True)
-euv = pysat.Instrument('icon', 'euv', sat_id='', tag='level_2',
-                       clean_level='clean', update_files=True)
-fuv = pysat.Instrument('icon', 'fuv', sat_id='', tag='level_2',
-                       clean_level='clean', update_files=True)
-mighti_green = pysat.Instrument('icon', 'mighti', sat_id='green',
-                                tag='level_2', clean_level='clean',
-                                update_files=True)
-mighti_red = pysat.Instrument('icon', 'mighti', sat_id='red', tag='level_2',
-                              clean_level='clean', update_files=True)
+import pysat
+
+from pysatNASA.instruments import icon_ivm, icon_euv, icon_fuv, icon_mighti
+
+instruments = list()
 
-instruments = [ivm, euv, fuv, mighti_green, mighti_red]
+for inst_mod in [icon_ivm, icon_euv, icon_fuv, icon_mighti]:
```

Comment:
This loads everything everything into ICON, including line-of-sight vector wind data. While useful for someone who wants to dig into mighti processing, I would imagine the majority of users would confine themselves to the combined vector wind product.
How about for MIGHTI we only include the vector winds and temperature?

Reply:
updated